test/integration/benchmark/jsonify/main.go

```diff
@@ -24,9 +24,36 @@ import (
 	"os"
 
 	benchparse "golang.org/x/tools/benchmark/parse"
-	"k8s.io/kubernetes/test/e2e/perftype"
 )
 
+// TODO(random-liu): Replace this with prometheus' data model.
```

Comment:
an almost identical version of this already exists in test/integration/scheduler_perf... can we limit ourselves to a single copy of these structs in each test area? what is consuming the output of these?

cc @kubernetes/sig-scalability-pr-reviews

Reply:
@liggitt  thanks for sharing this. I can move the `DataItem` struct to test/integration/util if it makes sense. Waiting for more comments from reviewers.

Reply:
> what is consuming the output of these?

This file has been merged with https://github.com/kubernetes/kubernetes/pull/59800
According to the corresponding issue https://github.com/kubernetes/kubernetes/issues/58154 the output seems to be consumed with http://perf-dash.k8s.io/#/?jobname=containerd-node-throughput&metriccategoryname=E2E&metricname=PodStartup&Metric=create_to_schedule I guess.

It is nice to get feedback from the author for sure.
/cc @porridge

Reply:
Hm, it's been ages since I looked at that codebase. I guess perfdash does not really care as long as it can import these structs from *somewhere*, but I'll let its current maintainers speak up.